### PR TITLE
Wait time update

### PR DESCRIPTION
**Pull request by itssumertime**
Addresses part of #166 by allowing minutes in the 45-59 range to show specifically rather than round to "1 hour". This solves the "1 hour to 1 hour" case to start including minutes for the first number. However, it does not fix "1 hour 15 mins" showing up correctly (still rounds to "1 hour").

Adding test for this incremental fix.

Test still doesn't pull in changes directly from index.js file.

### CODE REVIEW

File: callmeback/frontend/src/index.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ moment.updateLocale('en', {
     h: '1 hour',
   },
 });
+// Specify exact minutes in the 1-59 range. Without this, 45-59 mins is
+// rendered as "1 hour".
+moment.relativeTimeThreshold('m', 59);
```
Comment (egracer) [inline callmeback/frontend/src/index.js:19]: Depending on how big this `moment` configuration becomes, might make sense to pull into a separate file.

Comment (mscharfstein) [inline callmeback/frontend/src/index.js:19]: does this not also fix 1 hr 15 min rounding to 1 hour?

Comment (itssumertime) [reply callmeback/frontend/src/index.js:19]: Agreed. Hopefully it ends up only being one more line for the final fix.

Comment (itssumertime) [reply callmeback/frontend/src/index.js:19]: Not in my tests (and I wasn't able to get this time in the full server to try it).

**Pull request merged**